### PR TITLE
add korean(ko) localization

### PR DESCRIPTION
**Pull request by dubh3**
add korean localization

```json
{
  "@@locale": "ko",
  "backButtonTooltip": "뒤로",
  "closeButtonLabel": "닫기",
  "searchLabel": "검색",
  "closeNavigationTooltip": "네비게이션 닫기",
  "openNavigationTooltip": "네비게이션 열기",
  "clickToSearch": "검색",
  "modalBarrierDismissLabel": "취소",
  "minimizeWindowTooltip": "최소화",
  "restoreWindowTooltip": "이전크기",
  "closeWindowTooltip": "창닫기",
  "dialogLabel": "대화상자",
  "cutActionLabel": "잘라내기",
  "copyActionLabel": "복사",
  "pasteActionLabel": "붙여넣기",
  "selectAllActionLabel": "전체선택",
  "newTabLabel": "새탭",
  "closeTabLabelSuffix": "탭닫기",
  "scrollTabBackwardLabel": "이전탭",
  "scrollTabForwardLabel": "다음탭",
  "noResultsFoundLabel": "결과없음",
  "copyActionTooltip": "클립보드로 복사",
  "cutActionTooltip": "클립보드로 잘라내기",
  "pasteActionTooltip": "클립보드로 붙여넣기",
  "selectAllActionTooltip": "전체선택"
}
```

## Pre-launch Checklist

- [ ] I have updated `CHANGELOG.md` with my changes  
- [ ] I have run "optimize/organize imports" on all changed files
- [ ] I have added/updated relevant documentation

### CODE REVIEW

**Review by henry2man — APPROVED** [review]
Giving the fact that I don't know Korean😅, this LGTM!

File: lib/src/localization.dart [hunk 1]
```diff
@@ -196,16 +197,13 @@ class DefaultFluentLocalizations extends S implements FluentLocalizations {
   /// This method is typically used to create a [LocalizationsDelegate].
   /// The [FluentApp] does so by default.
   static Future<FluentLocalizations> load(Locale locale) {
-    return SynchronousFuture<FluentLocalizations>(
-        DefaultFluentLocalizations._defaultFluentLocalizations(locale));
+    return SynchronousFuture<FluentLocalizations>(DefaultFluentLocalizations._defaultFluentLocalizations(locale));
```
Comment (WinXaito) [inline lib/src/localization.dart:200]: Check formatting

File: lib/src/localization.dart [hunk 2]
```diff
@@ -196,16 +197,13 @@ class DefaultFluentLocalizations extends S implements FluentLocalizations {
...
   }
 
-  static const LocalizationsDelegate<FluentLocalizations> delegate =
-      _FluentLocalizationsDelegate();
+  static const LocalizationsDelegate<FluentLocalizations> delegate = _FluentLocalizationsDelegate();
```
Comment (WinXaito) [inline lib/src/localization.dart:203]: Check formatting

**Review by WinXaito — COMMENTED** [review]
I don't know Korean too, so ok to me 😄

**Comment by henry2man** [comment]
@dubh3 Please also add Korean in README.md supported languages list

**Review by henry2man — COMMENTED** [review]
Please also update README.md supported languages list

**Comment by rk0cc** [comment]
@bdlukaa please review this pull request since it opened almost 2 months ago and still not merged yet.

**Comment by bdlukaa** [comment]
@rk0cc tests are failing. i cant merge while tests are failing

**Pull request merged**